Project proposal 2/17

KamdenE opened the pull request:
This is our Project Proposal submission for Team 1.

KamdenE committed:
Merge pull request #8 from TnTech-ECE/main

Update Project Branch

KamdenE committed:
Added PDF Project Proposal

KamdenE committed:
Added markdown file of Project Proposal

KamdenE committed:
Delete Project Proposal/CIRCE Project Proposal copy.md

KamdenE committed:
Update Project Proposal to MD

KamdenE committed:
Rename Project Proposal to Project Proposal.md

KamdenE committed:
Fixed Budget format in Project Proposal.md

KamdenE committed:
Update Project Proposal.md

KamdenE committed:
Update Project Proposal.md

KamdenE commented:
Hey guys,

If you are reviewing the md file, please take a look at the downloadable pdf and rate it as well.

Thanks.

dtran143 reviewed (commented):
Well-rounded proposal. The sections do seem a bit long-winded with a lot of information being repeated. Some removal of redundant information could help with clarity. There seems to be a mention of lidar and visual based navigation systems but lacks a lot of depth in terms of implementation for the cable robot. More information can be elaborated on for these navigation algorithms.

tylerbare0 commented:
At first glance, the project proposal looks cumbersome and overly dense, a possible solution may be to break the paragraphs up into smaller subsections or use bullet points/number lists for certain sections such as the shall statements. As for the material, I found the tentative expenses for construction chart useful for organization and regret not putting one on our project proposal. Also there needs to be a Gantt chart for the timeline section of the report.

EWinnie commented:
Just a heads up to everyone reviewing. We understand that the mark down file is not pretty. If you don't care to take a look at the PDF you will see that it is where our effort went. Thanks for the review!

ellerenfroe commented:
The project covered all necessary areas. I do think adding some space for bullet points or numbered lists will help with the flow and understanding. Some of the information throughout seemed repetitive. The shall statements are very specific. Also, the report needs to be in third person.

ambacon43 commented:
The introduction of the project is very well done. I didn't fully understand the purpose of the project when it was explained in class, but after reading the first few paragraphs of the proposal, I felt like I had a good grasp on what the robot would do and why it would be beneficial. Overall, the proposal is very clearly laid out and explained. The only thing left fuzzy to me is how you are distinguishing "laying the cable" from "dragging the cable." Laying implies more intentionality, but what does this look like practically? Also, I do think it would be helpful to cut out some of the information that gets repeated, in particular in the formulating the problem and background sections, where the parts about dragging the cable and the robot being autonomous are almost verbatim. As a whole, I think the project proposal is looking really good so far, nice work!

Henry100202 reviewed (approved):
Honestly it is great over all but I see some terms and specifics are repeated. I love how detailed it is but keep in mind its fine to be a little vague so you don't lock yourself into something you might change later. Overall, I think your good though some things can be simplified down, but that would get rid of some detail and make it shorter.

93marisol commented:
I liked it. After reading it, I understood your project much better. The only suggestion: while reading the "measure of success" section, I got a little lost when you mentioned "the president's lawn" maybe go for "a designated testing area" or something to that effect. Great job.

krizithy reviewed (commented):
Crazy good proposal, but pretty wordy. If this info could be condensed somehow I think it would go a long way towards improving readability. Looking forward to seeing how this project turns out. o7

gwmarcrum committed:
Update Project Proposal.md

Deleted a redundant portion from the background section.

gwmarcrum committed:
Update Project Proposal.md

Indented paragraphs

EWinnie committed:
Update Project Proposal.md

Here I condensed my sections and cut the redundant parts. I also made it look pretty

crgraves8 committed:
Update Project Proposal.md

I corrected the indentations and cut out some of the repeated or unnecessary information in the section "existing resources."

macky-mack commented:
Overall the first draft was a great start and I was able to get a good understanding on the whole concept to be done. The chart as well was great and easy to read and know what each part needs to budget for.

CooperM42 committed:
Update Project Proposal.md

Edited the format of constraints and personnel.

crgraves8 committed:
Update Project Proposal.md

Just added indentations and white space in the specific implications section

KamdenE committed:
Update Project Proposal.md

Fixed Specifications, Budget, and Timeline with indentions.

KamdenE committed:
Update Project Proposal.md

Added lines between sections.

KamdenE committed:
Added 1st Rev of Team 1's Gantt chart

dtran143 commented:
Awesome proposal, some editing for visual clarity would be nice to have but the content is good. An example would be in SURVEY OF EXISTING RESOURCES that has extra-large spacing. MEASURES OF SUCCESS have indentations that make the sentences feel a bit off. Lastly, STATEMENT OF CONTRIBUTIONS would benefit from spacing and bullet points. I personally think that indentation is unnecessary for something written in markdown but that's only opinion.

Henry100202 reviewed (approved):
Honestly this looks amazing. Mark down is really annoying with bullet points and such, so I can see where you all tried to add them. I understand the project fully and its nice to read.

jmdavis43 reviewed (approved):
The project proposal looks great. I have no major concerns my only recommendation is to possibly shorten the survey of existing solutions, but you absolutely don't have to do this it just may help with readability.

macky-mack commented:
Overall, I do not see anything major that sticks out. You all added shall statements and made it sound cohesive with every section. I agree that the survey of existing solution is very long and could be shorten just a little bit, but it gave more information about the whole thing.

krizithy reviewed (approved):
Good stuff. The readability has improved since the previous iteration. As said above, some sections I feel could be trimmed--most notably the survey section--but hey I still think it's a great proposal

93marisol commented:
The formatting makes sense if your look at the PDF. Formulating the problem and time line have a lot of "we" other than that you already have my approval.

Jdcaten42 commented:
Well formatted and clear work!
A few notable problems: some grammar issues.

Specifications paragraph one: the plural of radius is radii - you had it correct later in the document which is why I was concerned you had it wrong here.

Survey of existing solutions: there are a bunch of random tabs in the paragraph suggesting that it was meant to be several smaller paragraphs rather than the single large mesh you have there. 

I love the look of that graph on budget, why is the battery element in there cubed?

Specific implications: Cost efficiency and reduced risk - same tabbing issue as is present in specifications.

I am worried about your sources...
Source 1 http://www.controleng.com/industrial-robot-safety-considerations- feels completely irrelevant to your article.

Source 2 https://www.armyupress.army.mil/Journals/NCO-Journal/Archives/2024/February/The-Ethics-of-Robots-in-War/ cannot be reached with the URL you provided (don't add spaces to URLs to make them look nicer) and is weak for your argument.

Source 3 https://www.unmannedsystemstechnology.com/2023/03/new-tethered-drone-designed-for-tactical-communications/ cannot be reached with the URL you provided. Same issue as above. It is a short document.

Source 4 https://www.ll.mit.edu/r-d/projects/balloon-based-resilient-communications cannot be reached with the URL you provided. Same issue as above. It is also a short document.

Source 5 https://www.uavos.com/products/fixed-wing-uavs/borey-10/ cannot be reached with the URL you provided. Same issue as above. It is a specification sheet.

Sources 6-9 - no problems at all. Thank you for including them.

Source 10 https://www.controleng.com/industrial-robot-safety-considerations-standards-and-best-practices-to-consider/ The link Works! It is an okay source for your material; however, it is limited in that it deals with industrial rather than military type autonomous robots. 

You likely could get rid of source 1 without any issues in your text.
You likely could get rid of sources 3-5 and shorten the survey of existing solutions by talking about the solutions in general.

I would have liked to see some resources pertaining more to your project: you are being sponsored by DEVCOM to develop this. They likely have some literature for you to use on your background and it would likely help you understand what this is and why they want it.

EWinnie committed:
Update Project Proposal.md

Fixing grammar in formulating the problem

m7md023x reviewed (commented):
It was clear and well informed, there is nothing to my knowledge i could say it's wrong or not clear.

MaddoxCagle commented:
I read the markdown file because it had a updated timestamp and the pdf's timestamp said last week. With that being said, I know the formatting is messed up on the markdown because we were supposed to look at the pdf. However, I do not know why it was showing that it hasn't been updated unless you all just left it the same. Regardless, the content was great, a little dense, but great. Your team was very thorough, and I believe you understand the problem and a possible solution. I do believe you all should do the next report/document in the markdown file completely so that there is no confusion.

crgraves8 committed:
Update Project Proposal.md

crgraves8 committed:
Update Project Proposal.md

Deleted conclusion paragraph under specific implications.

crgraves8 committed:
Update Project Proposal.md

Trimmed up "existing resources" section.

EWinnie committed:
Update Project Proposal.md

Clarification and summarizing of my parts as well as some formatting across the board

ellerenfroe commented:
The proposal has improved since the last time I read it! There are still areas that need to be in third person. Using words like we and our are not third person. For example in the timeline section, instead of saying "For our capstone project, we have this semester to design a self-autonomous cable-deploying robot." try saying "This semester the team has to design a self-autonomous cable-deploying robot". Other than that, it looks great!

EWinnie commented:
Please do not leave a review until the request has been made. We are not finished with revisions. Thank you and sorry for the inconvenience!

KamdenE committed:
Update Project Proposal.md

Updated Timeline to be in third person.

KamdenE commented:
We do appreciate GitHub reviews and comments allowing us to fix areas in our project but we kindly ask that you **do not review using the link in iLearn** as we have not officially requested a review yet. Thank you.

KamdenE committed:
Updated Personnel Section in Project Proposal.md

Fixed formatting as well as third person.

CooperM42 committed:
Update Project Proposal.md

Edited Personnel section.

CooperM42 committed:
Update Project Proposal.md

Edited the constraints section. Changed a "will" to a "shall".

gwmarcrum committed:
Update Project Proposal.md

EWinnie committed:
Update Project Proposal.md

Fixing third person issues and formatting.

KamdenE committed:
Updated Statement of Contributions in Project Proposal.md

KamdenE committed:
Updated Specifications in Project Proposal.md

KamdenE committed:
Updated budget in Project Proposal.md

KamdenE committed:
Added png version of Gantt chart.

KamdenE committed:
Updated Timeline in Project Proposal.md

KamdenE committed:
Updated Specifications in Project Proposal.md

BretBall commented:
Introduction - good
Formulating the problem - good
Background - good
Specifications - Why is (Review) there? Otherwise good
Constraints - Indentions seem off. Seems a little empty, but I'm a tweaker.
SOES - Survey of Existing Solutions not "Resources" (may not matter idk).
MOS - good
Resources - good
Budget - love the graph (I stole your all's idea)
Personnel - good - Proficient in troubleshooting. I would hope lol. Might want to list more programming as this project seems  programming heavy.
Timeline - good
Specific Implications - good - chat gpt fo sho.
Broader imp - good - Could also mention if a soldier dies laying cable that's another 500,000 down the drain for life insurance.
SOC - good
Resources - I'm not looking at those lol

This proposal is bangin. Everything I said can most likely be ignored.

ambacon43 commented:
Great job with the revisions! It flows a lot better now, with the sections being well-linked but not repetitive, which is hard to accomplish. Just a couple minor notes. 1) In the broader implications section you have an "our"- otherwise it seems to be perfectly in third person. 2) You may want to elaborate a bit in the constraints section on what the standards mentioned require and how you will comply, as well as on what you mean by "it won't negatively impact the general public," which is a little vague. Otherwise it seems to me to be the right balance of simple & technical explanation, nice work!

KamdenE committed:
Updated Broader Implications in Project Proposal.md

Fixed for third person correction.

KamdenE committed:
Updated Constraints in Project Proposal.md

Fixed to be more elaborative and understanding.

tylerbare0 commented:
Overall I think the proposal is very clear and explains the project well. It might help to add more details about how CIRCEBOT will avoid moving obstacles, especially when GPS isn’t working (that may be a conceptual design topic though). I would also suggest how you will comply with some of the constraints because Micah will likely comment the same thing like he did on our proposal.

Hammy2017 reviewed (approved):
I think you guys have a very well written and concise proposal good job

dtran143 commented:
Good proposal, I liked the revamped wording for most of the sections. However, some sections that can be improved are measures of success and constraints. In measures of success, smaller bullet points talking about the specification compliance can help the reader understand the specific specification that you are mentioning. Additionally, the constraints can be elaborated on for the specification standard numbers just mentioning OSHA standards does not accomplish this.

KamdenE committed:
Updated Constraints in Project Proposal.md

Expanded on the "OSHA" regulations that were "transparent" to the readers. Elaborated on which constraints the team would use as guidelines for the robot.

krizithy reviewed (approved):
Good improvements over the previous iterations. The doc feels more consise and to the point, and after talking to members of the team, I recognize your proposal needs to contain more information. Something to include could be more specific standards and references. Micah will call you out for not referencing specific standards by name,

m7md023x reviewed (commented):
the proposal is good over all, but in specification you mentioned the device shall run at least 20 minutes. I think 20 minutes is very low for the minimum time.

KamdenE commented:
> the proposal is good over all, but in specification you mentioned the device shall run at least 20 minutes. I think 20 minutes is very low for the minimum time.

This specification comes from our customer **DEVCOM**. This is not a specification that we can change freely as that would cause significant redesign for our battery and cause a price increase applied to our budget. In short, 20 minutes is sufficient as it was what was required by our customer and we were told to base it off of that.

Henry100202 reviewed (approved):
Approved. Watch out for specific standards. Looks great now.

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,421 @@
+allow for easy reloading and quick replacement of Ethernet
+cable reels within 2 minutes, without the use of external tools.
+CirceBot shall use minor obstacle avoidance to avoid
+collisions.
```
Give specifications as bullet points to help readability. Also specifically list the requirements of your customer here. For example do not just say "including proper curve radii, strain relief, and tension tolerances" instead tell me what the proper radii is and what the max tension would be. Also how would you define "minor obstacle avoidance"... seems vague?

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,421 @@
+___
+#### V. CONSTRAINTS
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The material cost of CirceBot shall not exceed the outlined
+budget, and any excess funds shall remain or be returned to
```
Put the max dollar amount here.

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,421 @@
+budget, and any excess funds shall remain or be returned to
+Tennessee Tech University.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CirceBot shall comply with relevant safety standards, including specific OSHA standards such as OSHA 29 CFR 1910 Subpart O (Machinery and Machine Guarding) and OSHA 29 CFR 1910.212 (General requirements for all machines). Additionally, compliance with ANSI/RIA R15.06 (Industrial Robots and Robot Systems - Safety Requirements) and ISO 10218 (Safety Requirements  ...
```
Too general. Try to identify what specific implications the standards you mention have for your project.

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,421 @@
+___
+#### V. CONSTRAINTS
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The material cost of CirceBot shall not exceed the outlined
+budget, and any excess funds shall remain or be returned to
+Tennessee Tech University.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CirceBot shall comply with relevant safety standards, including specific OSHA standards such as OSHA 29 CFR 1910 Subpart O (Machinery and Machine Guarding) and OSHA 29 CFR 1910.212 (General requirements for all machines). Additionally, compliance with ANSI/RIA R15.06 (Industrial Robots and Robot Systems - Safety Requirements) and ISO 10218 (Safety Requirements  ...
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The manufacturing, testing, and implementation of CirceBot shall be conducted in a manner that ensures no harm or disruption to the general public on Tennessee Tech University’s campus or the surrounding Cookeville, TN area. This includes measures to prevent excessive noise, emissions, physical hazards, or interference with pedestrian and vehicular traffic.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Additionally, the design and construction of CirceBot shall align with any supplementary standards or requirements specified by the client or campus authorities to ensure operational safety and regulatory compliance.
```
Put specifications in bullet point list.. this will be more clear.

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,421 @@
+___
+#### VI. SURVEY OF EXISTING RESOURCES
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Existing solutions to this problem could be but are not
+limited to, autonomous UAVs (Unmanned Aerial Vehicle), Tethered drones, and High-Altitude 
+Loitering Relay (HAPS or Balloon-Based), yet none fully meet the CirceBot’s specifications. 
+For instance, tethered drones like the Orion Heavy Lift support variable-height antennas to 
+extend secure mobile networks by relaying communications between centralized command and contested 
+assets. Although capable of carrying a 4 kg payload at about 90 meters for up to 50 hours [3], they are 
+built for aerial relaying rather than continuous cable deployment.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;High-altitude balloons offer a longer-lasting alternative 
+as beyond-line-of-sight communication nodes when satellite links fall short. A helium-filled 
+weather balloon can ascend to 100,000 feet and cover over 600 miles in diameter. Although these 
+balloons offer impressive coverage and affordability, they are not designed for precise, on-demand 
+cable deployment. Their operation is heavily influenced by weather conditions, and they lack the 
+capability to navigate rugged, GPS-denied terrain [4].
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Unmanned aerial vehicles, like the BOREY 20 Fixed-Wing UAV, can 
+deploy lightweight Ethernet cables using advanced navigation methods such as LiDAR. Nevertheless, 
+their dependence on preprogrammed flight paths and GPS, along with imprecise cable management, creates 
+challenges for proper Ethernet installation. This project addresses these issues through a dedicated, 
+hardwired approach [5].
```
Help me understand which specific constrains / specifications are not met by alternatives. You could number your specifications and constraints to make them easier to reference.

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,421 @@
+___
+#### XI. TIMELINE
+
+![Gantt Chart](https://github.com/TnTech-ECE/S25_Team1_MyCapstoneProject/blob/Project-Proposal/Project%20Proposal/Team_1_Gantt_Chart.png)
```
Next semester we will build and test you design. In your Gannt chart, plan for next semester as well.. use approximate dates.

EWinnie replied:
We are still working on the gantt chart.

EWinnie committed:
Update Project Proposal.md

specifications per micahs comments

EWinnie committed:
Update Project Proposal.md

CooperM42 committed:
Update Project Proposal.md

Converted constraints into a bullet point list.

KamdenE committed:
Update Specifications in Project Proposal.md

Added bullet points now.

crgraves8 committed:
Update Project Proposal.md

Edited the "Survey of Existing Resources"  section

CooperM42 committed:
Update Project Proposal.md

Added more specific constraints from the conceptual design.

KamdenE committed:
Added second Gantt chart for Fall semester

KamdenE committed:
Delete Project Proposal/Team_1_Gantt_Chart.png

KamdenE committed:
Added revision of 1st semester Gantt chart

KamdenE committed:
Updated Timeline in Project Proposal.md

Added Gantt charts and revisions to the paragraph wording.

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,438 @@
+budget, and any excess funds shall remain or be returned to
+Tennessee Tech University.
+
+* CirceBot shall comply with relevant safety standards, including specific OSHA standards such as OSHA 29 CFR 1910 Subpart O (Machinery and Machine Guarding) and OSHA 29 CFR 1910.212 (General requirements for all machines). Additionally, compliance with ANSI/RIA R15.06 (Industrial Robots and Robot Systems - Safety Requirements) and ISO 10218 (Safety Requirements for Industrial Robots) will ensure ...
+
+  - ANSI/TIA-568.2-D: This standard specifies that for unshielded twisted-pair (UTP) cables, the minimum bend radius should be four times the cable's outer diameter.
+ 
+  - ANSI/TIA-568.2-D: This standard emphasizes the importance of proper strain relief to prevent mechanical stress on cables, which can lead to performance degradation or failure. Implementing strain relief mechanisms, such as boots or bushings, helps maintain the integrity of the cable by preventing excessive bending and pulling forces.
+ 
+  - ANSI/TIA-568.2-D: This standard provides guidelines on the maximum pulling tension for Ethernet cables during installation. While specific tension tolerances can vary based on the cable manufacturer and type, a general guideline is to avoid exceeding a pulling tension of 25 pounds (110 Newtons) during installation. Exceeding this tension can cause stretching or damage to the cable, leading to ...
+ 
+  - ANSI/TIA-568.2-D: This standard specifies the maximum allowable length for twisted-pair Ethernet cables in standard networking applications. The maximum length is defined as 109 yards (328 feet or approximately 100 meters), beyond which signal attenuation, latency, and packet loss can degrade network performance. To extend connectivity beyond this limit, additional hardware such as repeaters, ...
+ 
+  - ANSI/RIA R15.08: This standard establishes safety requirements for autonomous mobile robots (AMRs), including obstacle detection and avoidance. It defines acceptable sensor technologies, response times, and stopping distances to ensure safe operation in dynamic environments.
```
Much better. Some of these are not formulated as shall statements. Reformulate them as shall statements.

micahr234 commented on Project Proposal/Project Proposal.md:
```diff
@@ -0,0 +1,438 @@
+___
+#### IV. SPECIFICATIONS
+# CirceBot Specifications
+
```
I think DEVCOM specified that the robot has to operate in an RF contested environment. Seems like this would be an important spec to include.

EWinnie committed:
Update Project Proposal.md

Second round of Mr. Rentschler review.
